lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -355,7 +357,7 @@ protected function _getPortableTableColumnDefinition($tableColumn)
         $jsonb     = null;
 
         $dbType = strtolower($tableColumn['type']);
-        if (strlen($tableColumn['domain_type']) && ! $this->_platform->hasDoctrineTypeMappingFor($tableColumn['type'])) {
+        if (strlen((string) $tableColumn['domain_type']) && ! $this->_platform->hasDoctrineTypeMappingFor($tableColumn['type'])) {
```

Comment:
A non-string would not pass this check. No need to cast and then check.

Reply:
changing to `!== null`